extras/vocabs.yml

```
@@ -5,3 +5,34 @@ vocabs:
   rightsstatements:
     dereferenceable_ontology: http://rightsstatements.org/vocab/1.0/
     label_property: http://www.w3.org/2004/02/skos/core#prefLabel
+  wikidata:
+    dereferenceable_ontology: https://query.wikidata.org/
+    label_property: http://www.w3.org/2004/02/skos/core#prefLabel
+  ulan:
+    dereferenceable_ontology: http://vocab.getty.edu/ulan/
+    label_property: http://www.w3.org/2004/02/skos/core#prefLabel
+  aat:
+    dereferenceable_ontology: http://vocab.getty.edu/aat/
+    label_property: http://www.w3.org/2004/02/skos/core#prefLabel
+  lcgft:
+    dereferenceable_ontology: http://id.loc.gov/authorities/genreForms
+    label_property: http://www.w3.org/2004/02/skos/core#prefLabel
+  fast:
+    dereferenceable_ontology: http://id.worldcat.org/fast/ontology/1.0/#fast
+    label_property: http://schema.org/name
+  geonames:
+    dereferenceable_ontology: https://www.geonames.org/ontology
+    label_property: http://www.geonames.org/ontology#name
+  lcsh:
+    dereferenceable_ontology: http://id.loc.gov/authorities/subjects
+    label_property: http://www.w3.org/2004/02/skos/core#prefLabel
+  tgm:
+    dereferenceable_ontology: http://id.loc.gov/vocabulary/graphicMaterials
+    label_property: http://www.w3.org/2004/02/skos/core#prefLabel
+  iso639-2b:
+    dereferenceable_ontology: http://id.loc.gov/vocabulary/iso639-2
+    label_property: http://www.w3.org/2004/02/skos/core#prefLabel
+  creativecommons:
+    dereferenceable_ontology: https://api.creativecommons.org/rest/1.5/details?license-uri=
+    label_property: http://www.w3.org/2004/02/skos/core#prefLabel
```

Where are you getting this from?  Shouldn't it be `http://creativecommons.org/ns#license-name`?

```suggestion
    label_property: http://creativecommons.org/ns#license-name
```

---

@markpbaggett - this is addressed, thanks.